src/test/regress/expected/multi_materialized_view.out

```diff
@@ -0,0 +1,125 @@
+---
+--- MULTI_MATERIALIZED_VIEW
+---
+-- This file contains test cases for materialized view support. It verifies
+-- various Citus features: 
+-- materialized views work
+-- insert into... select works with views
+CREATE VIEW air_shipped_lineitems AS SELECT * FROM lineitem_hash_part WHERE l_shipmode = 'AIR';
```

Comment:
In newer test files we usually do something like: 
```sql
CREATE SCHEMA multi_materialized_view;
SET search_path TO multi_materialized_view;
...
DROP SCHEMA multi_materialized_view CASCADE;
``` 

This ensures the test doesn't leave stuff behind, makes sure the names don't collide with other tests, and also helps detect schema-related issues.